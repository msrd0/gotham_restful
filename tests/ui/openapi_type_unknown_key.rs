#[macro_use] extern crate gotham_restful;

#[derive(OpenapiType)]
struct Foo
{
	#[openapi(like = "pizza")]
	bar : String
}

fn main()
{
}
