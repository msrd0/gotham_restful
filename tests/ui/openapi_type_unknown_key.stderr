error: Unknown key
 --> $DIR/openapi_type_unknown_key.rs:6:12
  |
6 |     #[openapi(like = "pizza")]
  |               ^^^^
